conda_forge_tick/git_utils.xsh

```diff
@@ -160,6 +161,24 @@ def refresh_pr(pr_json: LazyJson, gh=None):
         return pr_obj.as_dict()
 
 
+def ping_maintainers(pr_json: LazyJson, gh=None):
+    if gh is None:
+        gh = github3.login($USERNAME, $PASSWORD)
+    if not pr_json['state'] == 'closed':
+        r = requests.get(pr_json['statuses_url'],
+                         auth=($USERNAME, $PASSWORD))
+        statuses_json = r.json()
+        current_status = {i['context']: (i['state'], i['id']) for i in
+             sorted(statuses_json, key=lambda x: x['updated_at'])}
+        cached_status = LazyJson(pr_json['id'] + 'status.json')
+        # If the status has been updated, and none are pending
+        if current_status != cached_status and all(i['state'] != 'pending' for i in current_status):
+            pr_obj = github3.pulls.PullRequest(pr_json, gh)
+            if pr_obj.create_review_requests(team_reviewers=['conda-forge/' + pr_json['head']['repo']['name']]):
```

Comment:
I don't think this does what you want because the team itself as the reviewers, ie `@conda-forge/pkg-maintainers`.  Rather I think you want to add the list of maintainers here, not the team.